app/services/ArrivalNotificationService.scala

```diff
@@ -0,0 +1,60 @@
+/*
```

Comment:
Do you think we should move this into an ArrivalNotificationRepository? I'm just thinking we could have a separate service which calls the repository and keeps the controller and persistence layers apart. The service could be responsible for how to handle responses from mongo.